src/clr-angular/data/datagrid/datagrid-if-expanded.service.ts

```diff
@@ -33,7 +33,7 @@ export class DatagridIfExpandService extends IfExpandService {
     }
   }
 
-  private _replace: BehaviorSubject<boolean> = new BehaviorSubject(false);
+  private _replace: BehaviorSubject<boolean> = new BehaviorSubject(false as boolean);
```

Comment:
It was upset about this, and I don't know why. I had to force cast it because it was saying `false` is not of type `boolean` and seems like some TS silliness or RxJS generic implementation issue.

Reply:
Ah. Makes sense. Thanks!